src/sentry/testutils/factories.py

```diff
@@ -326,11 +328,11 @@ def create_team_membership(team, member=None, user=None, role=None):
                 organization=team.organization,
                 defaults={"role": "member"},
             )
-            if created:
-                member.outbox_for_create().drain_shard(max_updates_to_drain=10)
-                organizationmember_mapping_service.create_with_organization_member(
-                    org_member=member
-                )
+            # if created:
+            #     member.outbox_for_create().drain_shard(max_updates_to_drain=10)
+            #     organizationmember_mapping_service.create_with_organization_member(
+            #         org_member=member
+            #     )
```

Comment:
Should this be deleted instead of commented out?

Reply:
Fixing these yes